workshops/templates/workshops/event_edit_form.html

```diff
@@ -47,5 +41,14 @@ <h2 id="associated-tasks">Associated tasks</h2>
     {{ task_form.as_p }}
     <button type="submit" class="btn btn-primary" name="add">Add</button>
 </form>
-
 {% endblock %}
+
+{% block extrajs %}
+{{event_form.media}}
+{{task_form.media}}
+<script type="text/javascript">
+$(document).ready(function(){
+    $('#id_event-start,#id_event-end').datepicker();
```

Comment:
Please add `{format: "yyyy-mm-dd"}` here - otherwise this plugin refuses to work properly.